### DISCUSSION TITLE
Is PHPSpreadsheet a good fit for my use case

### DISCUSSION THREAD

**Discussion by razvdana** [PHPOffice/PhpSpreadsheet#4103] [Q&A]
Hi, I have some multi-sheet excel files which use some VLOOKUP formulas to load some images, example:

![IMG-20240719-WA0002](https://github.com/user-attachments/assets/3b0be402-a32d-45db-acc4-71095dae72e1)

So far I have simply tried to copy paste the data onto my php website but the images are not being displayed. Is PHPSpreadsheet able to help me copy all the data including images(considering vlookup)  ? Is there another step I would need to do, such as first upload the images to my hosting provider and use them as URLS (if possible) ?

LE: If i manage to display the images through VBA, can this be used by PHPSpreadsheet as well so the images are still being copied ? Instead of only seeing the URL behind it

**Comment by MarkBaker** [comment]
MS Excel has only recently introduced support for images in cells (it first appeared in Beta releases on 18th Jul 2023), and this isn't yet supported by PhpSpreadsheet (it's currently under investigation/development); nor is the `IMAGE()` function yet supported (also Work in Progress).

**Comment by oleibman** [comment]
I believe this was solved by PR #4677 in January. I will leave this discussion open for a while in order to give you the opportunity to discuss it, but I will close it in a few weeks if you don't have anything to add.
